doc/Requirements Specification/Parts/Data.lyx

```diff
@@ -0,0 +1,309 @@
+
+\begin_layout Standard
+In the following section, “the software” refers to the software a user wants
+ to analyse with Beagle.
+ The term refers not only to source code, but also its conceptional attributes,
+ like its purpose, structure and 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glslink{software architecture}{architecture}
+\end_layout
+
+\end_inset
+
+.
+ The software is excpected to be built using 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{component-based software architecture}
+\end_layout
+
+\end_inset
+
+.
+ This assumption is with little loss of generality, as any software 
+\end_layout
+
+\begin_layout Section
+Input
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/D10/"
+
+\end_inset
+
+ The software’s source code.
+ It must be written in Java and be compilable by the JDK installed on the
+ PC Beagle runs on.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/D20/"
+
+\end_inset
+
+ The software's 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glsuserii{component}
+\end_layout
+
+\end_inset
+
+ boundaries.
+ Beagle uses the model to determine 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{component}
+\end_layout
+
+\end_inset
+
+ boundaries.
+ The model's quality and accuracy thus have a direct impact on Beagle's
+ results' quality and accuracy.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/D30/"
+
+\end_inset
+
+ The software's 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glsuserii{component}
+\end_layout
+
+\end_inset
+
+ 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SEFF}
+\end_layout
+
+\end_inset
+
+, contained in the 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{PCM}
+\end_layout
+
+\end_inset
+
+ instance (
+\begin_inset CommandInset ref
+LatexCommand ref
+reference "/D20/"
+
+\end_inset
+
+).
+ Beagle uses the 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{SEFF}
+\end_layout
+
+\end_inset
+
+ information to determine 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+glspl{internal action}
+\end_layout
+
+\end_inset
+
+ and measurement points.
+ Their quality and accuracy thus have a direct impact on Beagle's results'
+ quality and accuracy.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset VSpace defskip
+\end_inset
+
+
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OD10/"
+
+\end_inset
+
+ User provided information about the software's parts to be analysed.
```

Comment:
As above, let’s discuss that on the data pull request. (As all parts will be improved)